ext-src/NexusExplorer.ts

```diff
@@ -196,6 +196,10 @@ export class NexusExplorer {
       "nexusExplorer.viewNode",
       (node: ComponentEntry) => this.viewNode(node)
     );
+      
+    vscode.workspace.createFileSystemWatcher('**/Pipfile.lock', false, false, false).onDidChange((event) => {
```

Comment:
This is rad. I think we might want to adopt this pattern once #145 lands (a major rework of everything, but something that will likely excite you).